extensions/wikia/Recirculation/services/CakeRelatedContentService.php

```diff
@@ -21,58 +21,58 @@ class CakeRelatedContentService {
 	 * @param $ignore
 	 * @return RecirculationContent[]
 	 */
-	public function getContentRelatedTo($title, $universeName=null, $limit=5, $ignore=null) {
+	public function getContentRelatedTo( $title, $universeName = null, $limit = 5, $ignore = null ) {
 		$items = [];
 
-		if (!$this->onValidWiki() || !$this->onValidPage($title)) {
+		if ( !$this->onValidWiki() || !$this->onValidPage( $title ) ) {
 			return $items;
 		}
 
 		$api = $this->relatedContentApi();
 
 		try {
-			$filteredRelatedContent = $api->getRelatedContentFromEntityName($title, $universeName, $limit + 1, "true");
-			if (is_null($filteredRelatedContent)) {
-				$this->warning("getRelatedContentFromEntityName failed to retrieve recommendations", [
+			$filteredRelatedContent = $api->getRelatedContentFromEntityName( $title, $universeName, $limit + 1, "true" );
+			if ( is_null( $filteredRelatedContent ) ) {
+				$this->warning( "getRelatedContentFromEntityName failed to retrieve recommendations", [
 						"title" => $title,
 						"limit" => $limit
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "limit" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)